### DISCUSSION TITLE
Hacktoberfest?

### DISCUSSION THREAD

**Discussion by Julian-B90** [stephanrauh/ngx-extended-pdf-viewer#2589] [General]
Would you like to take part in the Hacktoberfest?

**Comment by stephanrauh** [comment]
Sounds like an interesting event, and I'd like to encourage everybody to participate. @Julian-B90 Would you mind to add a few words what the Hacktoberfest is? And is it still possible to participate?

**Comment by Julian-B90** [comment]
Hacktoberfest is DigitalOcean’s annual event that encourages people to contribute to open source throughout October.

https://hacktoberfest.com/about/

You only have to do this to join in:

1. Register for Hacktoberfest: Visit the official Hacktoberfest website and log in with your GitHub account. https://hacktoberfest.com/auth/

2. Mark your repository: Make sure that your repository is public and add the hacktoberfest label to your issues and pull requests. This signals to participants that contributions to your project count for Hacktoberfest.
